C/plugins/north/OMF/plugin.cpp

```diff
@@ -348,7 +348,14 @@ const char *PLUGIN_DEFAULT_CONFIG_INFO = QUOTE(
 			"order": "30",
 			"group": "Formats & Types",
 			"displayName": "Complex Types"
-		}
+		},
+        "EnableOMFLogging" : {
+            "description" : "If true, write to the omf.log file. If false, do not create log entries.",
+            "type" : "boolean",
+            "default" : "false",
+            "order" : "31",
+            "displayName" : "Enable OMF Logging"
```

Comment:
Consistent with @MarkRiddoch 's comment, could this be "_Enable OMF Tracing_?"

Reply:
May be better to say,

Enable Tracing 
Or
Enable Debug Tracing

Repeating the OMF word is for some specific purpose?